modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultGenerator.java

```diff
@@ -892,7 +892,7 @@ private void generateSupportingFiles(List<File> files, Map<String, Object> bundl
 
             sb.append(System.lineSeparator());
 
-            System.err.println(sb.toString());
+            LOGGER.debug(sb.toString());
```

Comment:
@agilob thanks for the PR. I no longer get the NPE using the spec you provided 👍 

For this line, what about using `LOGGER.error` instead?

Reply:
replaced